Fix Github Light and Pitaya Smoothie contrast color bugs

steff456 opened the pull request:
Fixes #59 

This PR,

- [x] Fixes the issues found in the Pitaya Smoothie and Github Light themes.

Here are some screenshots on how the current themes look,

**Github Light**

- [x] The active tab colour is noticeable
- [x] Hover background (menu) has enough contrast against the background colour
- [x] Colours in the kernel/tab manager sidebar 
- [x] Active indicator on the search bar
- [x] Toggle background is noticeable in the status bar

<img width="1792" alt="image" src="https://github.com/Quansight-Labs/jupyterlab-accessible-themes/assets/20992645/cd134408-bf61-4949-b49c-c676af99d47c">

<img width="522" alt="image" src="https://github.com/Quansight-Labs/jupyterlab-accessible-themes/assets/20992645/063d5947-6362-4217-9126-ccc3e78b7a64">

<img width="323" alt="image" src="https://github.com/Quansight-Labs/jupyterlab-accessible-themes/assets/20992645/9a9010bd-814e-4085-a911-5c2ce9b91df4">

<img width="331" alt="image" src="https://github.com/Quansight-Labs/jupyterlab-accessible-themes/assets/20992645/d3e1c195-8b7a-467d-ad0e-e8cd7c74bd06">


**Pitaya Smoothie**

- [x] Icons have enough contrast against the background
- [x] Active indicator on the search bar

<img width="537" alt="image" src="https://github.com/Quansight-Labs/jupyterlab-accessible-themes/assets/20992645/1fdc94d8-cb67-4aa7-b309-c7dff694cf4d">

gabalafou reviewed (commented):
Great work, Stephannie! I understand that twiddling the CSS values is tricky, rinse-and-repeat kind of work! 

I just have one question. Does this PR fix all of the issues identified in the corresponding #59 or only some of them? I'm confused because some of the tasks in #59 are checked off but some are not.

steff456 commented:
This PR addresses all the issues in #59 for Github Light and Pitaya Smoothie. I made the changes of Github Dark in the PR #53.

Hope this clarifies!

gabalafou commented:
Sure, thanks! Who is going to be responsible for checking off the tasks in issue #59?

Maybe also the title of the PR should be renamed to "Fixes for Pitaya Smoothie and Github Light." I understand that it says this already in the description, but right now the PR title is really generic.

steff456 commented:
> Who is going to be responsible for checking off the tasks in issue https://github.com/Quansight-Labs/jupyterlab-accessible-themes/issues/59?

If we agree on the changes that I made for fixing them, I could check them once this PR is merged.

trallard commented:
Disagree - changes should be checked BEFORE merging not the other way around

trallard commented:
For example there is one that was not checked off for the dark theme as when I did the review I noticed the fix did not fix the colour contrast issue

trallard commented:
Or you provided you tested / verified the changes and they meet the contrast requirements. Which IMO is part of doing the fix.

steff456 commented:
I added the screenshots to the description of this PR, I will update it with the contrast that every pair has. Hope that helps with the review!

steff456 commented:
With the latest changes, here are the screenshots of every change with its contrast check. (I already updated the issue as well)

# Github Light

- [x] The active tab colour is noticeable. 

Blue line and text have enough contrast against the background color
<img width="699" alt="image" src="https://github.com/Quansight-Labs/jupyterlab-accessible-themes/assets/20992645/c4cd4ae9-d509-4339-a3b7-d219abf498fb">

- [x] Hover background (menu) has enough contrast against the background colour

Gray background with white text
<img width="913" alt="image" src="https://github.com/Quansight-Labs/jupyterlab-accessible-themes/assets/20992645/5d657ae7-42b0-431d-bc02-986a1902446e">

White background with gray hover color
<img width="931" alt="image" src="https://github.com/Quansight-Labs/jupyterlab-accessible-themes/assets/20992645/5c4529cf-529e-4164-a718-fefcfc05100d">

- [x] Colours in the kernel/tab manager sidebar 

<img width="736" alt="image" src="https://github.com/Quansight-Labs/jupyterlab-accessible-themes/assets/20992645/0d29b670-3227-40ad-8d9f-9383e7aae1d6">

- [x] Active indicator on the search bar

<img width="481" alt="image" src="https://github.com/Quansight-Labs/jupyterlab-accessible-themes/assets/20992645/56cfecec-8928-4014-a175-40c6cbb69cff">

<img width="266" alt="image" src="https://github.com/Quansight-Labs/jupyterlab-accessible-themes/assets/20992645/84908527-5071-4e30-ab58-b3f0028e3a66">

- [x] Toggle background is noticeable in the status bar

<img width="133" alt="image" src="https://github.com/Quansight-Labs/jupyterlab-accessible-themes/assets/20992645/2b138ae5-09cc-467e-8f96-1956f0ce7ec7">

Light gray background has enough contrast with toggle background
<img width="395" alt="image" src="https://github.com/Quansight-Labs/jupyterlab-accessible-themes/assets/20992645/5c05a0c9-2e3f-4a2c-bb60-1c230a2374d9">

Toggle has enough contrast 
<img width="394" alt="image" src="https://github.com/Quansight-Labs/jupyterlab-accessible-themes/assets/20992645/e7a7f9c1-05f4-4985-8a94-5ce56ffb1209">

trallard commented:
I am going to merge this as it seems ok after a quick review and I suppose users might raise issues later
